data/cleansed/payment-methods.json

```diff
@@ -623,6 +623,21 @@
     ],
     "capabilities": []
   },
+  {
+    "id": "twint",
+    "type": "online",
+    "name": "Twint",
+    "small_width": 65,
+    "small_height": 41,
+    "medium_width": 100,
+    "medium_height": 60,
+    "large_width": 256,
+    "large_height": 256,
+    "regions": [
+      "world"
```

Comment:
I think Twint is for Switzerland from the [ticket](https://flowio.atlassian.net/browse/PAY-3830), should we specify it in `regions`?

Reply:
Good point - following the pattern here. We are leaving it open to `world` in the reference but manage its actual availability in payment.